hamilton/graph.py

```diff
@@ -143,7 +143,7 @@ def update_dependencies(
 
 
 def create_function_graph(
```

Comment:
The type signature of this should be updated.

Reply:
also 🤦 (at myself I think) that the name of this function and what it returns - a dict, not a function graph...